poetry.lock

```diff
@@ -631,31 +631,28 @@ develop = false
 [package.dependencies]
 channels = {version = "*", optional = true, markers = "extra == \"channel-auth\""}
 cryptography = "*"
-Django = ">=4.2.16,<4.3.0"
+Django = ">=4.2.5,<4.3.0"
```

Comment:
This appears to not be picking up (reverting) https://github.com/ansible/django-ansible-base/pull/605 from Oct 22.